SPECS/git/git.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Fast distributed version control system
 Name:           git
-Version:        2.33.8
-Release:        2%{?dist}
+Version:        2.39.4
```

Comment:
Looks like 8 AMD64 (ARM64 is still going) ptests for packages building with `git` have now failed. Please take a look at them, to see what might be wrong.

Reply:
Hi @PawelWMS , 
9 out of AMD64 ptests had been failing in 2.0 as well, and
ostree ptest is [ignored ](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=573767&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=b85ceaf1-cf43-56e0-c161-c7d217bb7eb3&l=56) in main and 2.0. Should the packages in test_ignore_list be added to the github PR check as well to avoid the check blocking known ptest failures?
Hence, there are no AMD64 ptest regressions due to this PR.
Here's the ptest history of all the 10 failing AMD64 ptests
![image](https://github.com/microsoft/azurelinux/assets/33163197/7f4b81aa-c83e-4a3c-a58b-1d49c8146e60)
Hence there are no AMD64 ptest regressions due to the git upgrade

Reply:
similarly for ARM64

- 6 of the 8 ptests had been failing since past 5+ releases.
- ostree ptest is ignored
- bpftrace ptest is [ignored](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=573676&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=b85ceaf1-cf43-56e0-c161-c7d217bb7eb3&l=600) for ARM64. 
  Should the ptests that are skipped for ARM64 packages be skipped in the github PR check too?

Here's the ARM64 ptest history of these packages for the past few releases.
![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/70314e59-86d8-410d-a3ff-7a572b6e6066/28282382-a224-4a61-802d-a93e59d35fa9/Untitled.png)

Reply:
@PawelWMS
git is required to build 130+ packages all of which built successfully, and there are no ptest regressions due to the git version upgrade.

Reply:
Thank you for the investigation, @sindhu-karri! I've synced with @jslobodzian to get a better understanding of our policy for flaky ptests and in addition to looking at the test results, we also want to compare the build logs for failing tests from the PR with the ones from `main` to see if they didn't "regress more". Not a great check, I know, but the best we've got until we restore these tests to stable green.